react/features/base/config/reducer.ts

```diff
@@ -535,6 +535,30 @@ function _translateLegacyConfig(oldValue: IConfig) {
         };
     }
 
+    if (oldValue.autoKnockLobby !== undefined
+        && newValue.lobby?.autoKnockLobby === undefined) {
+        newValue.lobby = {
+            ...newValue.lobby,
+            autoKnockLobby: oldValue.autoKnockLobby
+        };
+    }
+
+    if (oldValue.enableLobbyChat !== undefined
+        && newValue.lobby?.enableLobbyChat === undefined) {
+        newValue.lobby = {
+            ...newValue.lobby,
```

Comment:
This would fail if it doesn't exist before. I'd suggest you firs create the config.lobby empty object if it doesn't exist.